### PR TITLE
CSS Task 1

### PR DESCRIPTION
**Pull request by dudzon**
https://dudzon.github.io/ui-internship-lwo2/css_tasks/task1/

### CODE REVIEW

File: css_tasks/task1/index.html [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+          <!-- Left Sidebar -->
+          <td width="198px" valign="top">
+            <table cellpadding="5px" width="100%">
+              <tbody>
+                <th height="40px" align="left">
+                  <font size="5" face="Montserrat">Menu</font>
+                </th>
+                <tr>
+                  <td>
+                    <font face="Montserrat"><a href="#">Main</a></font>
```
Comment (tsvetkovv) [inline css_tasks/task1/index.html]: Is it not enough setting font only for root element?

File: css_tasks/task1/index.html [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+            >Markup of "Basic HTML" Task</font
+          >
+        </th>
+      </thead>
+      <tbody>
+        <tr>
+          <!-- Left Sidebar -->
+          <td width="198px" valign="top">
+            <table cellpadding="5px" width="100%">
+              <tbody>
```
Comment (tsvetkovv) [inline css_tasks/task1/index.html]: why table is here? It is not necessary

File: css_tasks/task1/index.html [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+                          </select>
+                        </td>
+                      </tr>
+                    </table>
+                  </td>
+                </tr>
+                <!-- Checkbox -->
+                <tr>
+                  <td>
+                    <table cellpadding="5px">
```
Comment (tsvetkovv) [inline css_tasks/task1/index.html]: More tables for god of tables =) Why so many?

File: css_tasks/task1/index.html [hunk 4]
```diff
@@ -0,0 +1,273 @@
...
+          <!-- Right Sidebar -->
+          <td width="198px" valign="top">
+            <table width="100%">
+              <tbody>
+                <!-- Inputs -->
+                <tr>
+                  <td>
+                    <table width="100%" cellpadding="5px">
+                      <tr>
+                        <td><font face="Montserrat" size="3">Login:</font></td>
```
Comment (tsvetkovv) [inline css_tasks/task1/index.html]: login should be label element associated with input
and all inputs (button as well) should be inside `<form>` element

**Review by tsvetkovv — CHANGES_REQUESTED** [review]

**Comment by tsvetkovv** [comment]
Please remove task#2 from this PR

File: css_tasks/task1/index.html (hunk 1 above)
Comment (dudzon) [reply css_tasks/task1/index.html]: Hm.., probably it's stupid to admit, but I can't figure out how I can use font tag on root. Never used font tag before :)

File: css_tasks/task1/index.html (hunk 2 above)
Comment (dudzon) [reply css_tasks/task1/index.html]: I don't know how I can reset default padding on ul without styling:)

File: css_tasks/task1/index.html (hunk 3 above)
Comment (dudzon) [reply css_tasks/task1/index.html]: Yeah, it seems I was focused on including as many tables as I could :)

File: css_tasks/task1/index.html (hunk 4 above)
Comment (dudzon) [reply css_tasks/task1/index.html]: Done it.

**Review by tkachukivan — APPROVED** [review]
Do Viktor changes, and yes here have to be only 1 task :)

**Comment by tsvetkovv** [comment]
@dudzon why did you close the PR?

**Pull request closed without merging**